src/Uno.Extensions.sln

```diff
@@ -150,240 +150,788 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Uno.Extensions.Core.Generat
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Uno.Extensions.Core.Tests", "Uno.Extensions.Core.Tests\Uno.Extensions.Core.Tests.csproj", "{70345491-481C-4B75-902C-8D7E473EE4BA}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Uno.Extensions.Navigation.WinUI.Markup", "Uno.Extensions.Navigation.WinUI.Markup\Uno.Extensions.Navigation.WinUI.Markup.csproj", "{93D4147A-94BC-4599-94EB-51E77E673BF9}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Uno.Extensions.Navigation.WinUI.Markup", "Uno.Extensions.Navigation.UI.Markup\Uno.Extensions.Navigation.WinUI.Markup.csproj", "{93D4147A-94BC-4599-94EB-51E77E673BF9}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Uno.Extensions.Reactive.WinUI.Markup", "Uno.Extensions.Reactive.WinUI.Markup\Uno.Extensions.Reactive.WinUI.Markup.csproj", "{2890C1C1-D58C-4D39-963E-F27D17F96E9F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Uno.Extensions.Reactive.WinUI.Markup", "Uno.Extensions.Reactive.UI.Markup\Uno.Extensions.Reactive.WinUI.Markup.csproj", "{2890C1C1-D58C-4D39-963E-F27D17F96E9F}"
+EndProject
+Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Uno.Extensions.RuntimeTests.Shared", "Uno.Extensions.RuntimeTests\Uno.Extensions.RuntimeTests.Shared\Uno.Extensions.RuntimeTests.Shared.shproj", "{6279C845-92F8-4333-AB99-3D213163593C}"
```

Comment:
Can we avoid including tests in the main Uno.Extensions.sln to make it quicker to generate packages? We currently have some unit test projects in here but would be good for them to be moved out along with runtime tests into the testing folder and a different solution

Reply:
Strongly disagree: I would rather split extensions in separated solutions, but tests must stay close to the code.

Reply:
Well we could move the "RuntimeTest runner app" to a dedicated solution, but even then it would make it way more difficult to reach ...